### DISCUSSION TITLE
When will refreshing or a live dashboard [F9] be added?

### DISCUSSION THREAD

**Discussion by Reyyancli** [ppy/osu#30658] [Q&A]
Sometimes i when i want to play multi with friends but i dont see them on the overlay. In stable it automatically updates based on user online status but in lazer it seems to always stay the same after you open it once.

I think the game is at a point where this should've been added by now since alot of people have shifted to lazer. The last mention of this i saw was from late 2022. on a random side note i think the wiki overlay should also have a redesign as its a little claustrophobic

**Comment by peppy** [comment]
It's already live, but only shows currently playing users right now.

If you have friends on your friends list, you can switch to friends display mode, which will always show them even when offline. I guess your issue is that their status doesn't update in this display mode?

**Reply by kvietcong** [reply]
Yeah, the only way to get the current status to update is to restart the game. You can't spectate or invite to room when they're in an offline state so it's a bit annoying needing to restart every time you want to do either if you logged in before the target friend
